docker/docker-compose.yml

```diff
@@ -74,8 +74,31 @@ services:
         # service_started, not service_healthy as pinot and deps can take longer than 60s to start
         condition: service_started
      
+  hypertrace-alert-engine:
+    image: hypertrace/hypertrace-alert-engine:1.0.7
```

Comment:
Once, we have fixed for latest version, we will remove the version tag.

Reply:
Fixed! Could you check once and update here.

Reply:
Done